src/editor/components/modals/revision-diff.jsx

```diff
@@ -69,8 +123,18 @@ export function mapStateToProps(state) {
   return {
     currentRevisionContent,
     revisionContentFetchStatus,
-    previousRevisionContent
+    previousRevisionContent,
+    canRevert:
+      selectedRevisionId !== undefined &&
+      state.iomd !== currentRevisionContent &&
+      revisionContentFetchStatus === "IDLE",
+    date: revisionDate
```

Comment:
Let's just pass `selectedRevision` here, and do the formatting where it is actually required based on its `created` property.